Automattic/wp-calypso
# Post-checkout: Display thank you page after purchasing a Jetpack product

Currently, if you try to purchase a Jetpack product (visit http://wpcalypso.wordpress.com/checkout/:site/jetpack_backup_daily where `:site` is a Jetpack site slug), you'll get redirected to the Jetpack onboarding flow with the auto plugin setup.

This is unexpected - Jetpack product purchase should just lead to a "thank you" page.

#### Changes proposed in this Pull Request

* Products: Rename yearly Jetpack backup products constant
* Products: Add a generic Jetpack Backup products constant
* Checkout: Display thank you page after purchasing a Jetpack product.

#### Testing instructions

* Checkout this branch.
* Go to http://calypso.localhost:3000/checkout/:site/jetpack_backup_daily where `:site` is the slug of a connected Jetpack site.
* Finish the purchase.
* Verify you see a thank you page and a "Congratulations on your purchase! You will receive an email confirmation shortly." message.
* Now go to purchase a plan for Jetpack site that's currently on the free plan.
* Verify you still land on the "my plan" page and you see the security checklist with the plugin auto setup.
